sequence_test.go

```diff
@@ -69,3 +70,37 @@ func TestLocationParser(t *testing.T) {
 		t.Errorf("Feature sequence parser has changed on test 'complement(join(893..1098,1101..2770))'. Got this:\n%s instead of \n%s", featureComplementJoin, seqComplementJoin)
 	}
 }
+
+func ExampleAnonymizeSequence(t *testing.T) {
```

Comment:
remove `t *testing.T ` argument declaration. Isn't used in this kind of test and is breaking test run.